migrations/versions/56eff971505c_.py

```diff
@@ -0,0 +1,28 @@
+"""empty message
```

Comment:
Please add message here to make migrations more managable.
This could be as short as "remove country for node"